### PR TITLE
Sarah's portfolio

### PR DESCRIPTION
**Pull request by Kneedler**

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,418 @@
...
+
+  <body>
+    <header>
+      <nav class="nav-icons">
+        <a
+          href="https://www.linkedin.com/in/sarah-kneedler/"
+          class="nav-links"
+          target="blank"
+          alt="linkedin icon and link"
+        >
```
Comment (Petrasoderstrom1612) [inline code/index.html]: `a` tag does not support alt attribute, change instead to  `aria-label="visit my linked in" ` for blind users.

File: code/index.html [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+              d="M9,25H4V10h5V25z M6.501,8C5.118,8,4,6.879,4,5.499S5.12,3,6.501,3C7.879,3,9,4.121,9,5.499C9,6.879,7.879,8,6.501,8z M27,25h-4.807v-7.3c0-1.741-0.033-3.98-2.499-3.98c-2.503,0-2.888,1.896-2.888,3.854V25H12V9.989h4.614v2.051h0.065 c0.642-1.18,2.211-2.424,4.551-2.424c4.87,0,5.77,3.109,5.77,7.151C27,16.767,27,25,27,25z"
+            ></path>
+          </svg>
+        </a>
+        <a
+          href="https://github.com/Kneedler"
+          class="nav-links"
+          target="blank"
+          alt="github icon and link"
+        >
```
Comment (Petrasoderstrom1612) [inline code/index.html]: `a` tag does not support alt attribute, change instead to  `aria-label="visit my github" ` for blind users.

File: code/index.html [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+              d="M15,3C8.373,3,3,8.373,3,15c0,5.623,3.872,10.328,9.092,11.63C12.036,26.468,12,26.28,12,26.047v-2.051 c-0.487,0-1.303,0-1.508,0c-0.821,0-1.551-0.353-1.905-1.009c-0.393-0.729-0.461-1.844-1.435-2.526 c-0.289-0.227-0.069-0.486,0.264-0.451c0.615,0.174,1.125,0.596,1.605,1.222c0.478,0.627,0.703,0.769,1.596,0.769 c0.433,0,1.081-0.025,1.691-0.121c0.328-0.833,0.895-1.6,1.588-1.962c-3.996-0.411-5.903-2.399-5.903-5.098 c0-1.162,0.495-2.286,1.336-3.233C9.053,10.647,8.706,8.73,9.435,8c1.798,0,2.885,1.166,3.146,1.481C13.477,9.174,14.461,9,15.495,9 c1.036,0,2.024,0.174,2.922,0.483C18.675,9.17,19.763,8,21.565,8c0.732,0.731,0.381,2.656,0.102,3.594 c0.836,0.945,1.328,2.066,1.328,3.226c0,2.697-1.904,4.684-5.894,5.097C18.199,20.49,19,22.1,19,23.313v2.734 c0,0.104-0.023,0.179-0.035,0.268C23.641,24.676,27,20.236,27,15C27,8.373,21.627,3,15,3z"
+            ></path>
+          </svg>
+        </a>
+        <a
+          href="https://stackoverflow.com/users/19453494/kneedler"
+          class="nav-links"
+          target="blank"
+          alt="stackowerflow icon and link"
+        >
```
Comment (Petrasoderstrom1612) [inline code/index.html]: `a` tag does not support alt attribute, change instead to  `aria-label="visit my stackoverflow" ` for blind users.

File: code/index.html [hunk 4]
```diff
@@ -0,0 +1,418 @@
...
+            ></path>
+          </svg>
+        </a>
+      </nav>
+      <div class="header-info">
```
Comment (Petrasoderstrom1612) [inline code/index.html]: add tabindex="0" for screen readers as now the tab button jumps over

File: code/index.html [hunk 5]
```diff
@@ -0,0 +1,418 @@
...
+        <img
+          class="profile-img"
+          src="./assets/profilepic.webp"
+          alt="Photo of Sarah"
+        />
+      </div>
+    </header>
+
+    <section class="intro-wrapper">
+      <div class="intro-text">
```
Comment (Petrasoderstrom1612) [inline code/index.html:120]: add tabindex="0" for screen readers as now the tab button jumps over

File: code/index.html [hunk 6]
```diff
@@ -0,0 +1,418 @@
...
+          Node.js, Mongo DB, Web Accessibly, API:s, mob-programming,
+          pair-programming, Github.
+        </p>
+      </div>
+    </section>
+
+    <section class="projects-outer-wrapper">
+      <div class="inner-wrapper">
+        <div class="head-wrapper">
+          <h4>Featured projects</h4>
```
Comment (Petrasoderstrom1612) [inline code/index.html:148]: I also suggest to add the add tabindex="0" for screen in all `p` in this Features projects section

**Review by Petrasoderstrom1612 — COMMENTED** [review]
Fantastic work with the CSS! I love how you work with the hoover effects on the page and also the logos at the bottom. The page is neat and has really good contrast! Everything is centered and well placed on the page so the general user experience is fantastic. I would just suggest to go over the comments I made to fix some small details. Well done! You really created a beatiful portfolio and should be proud!

File: code/index.html (hunk 1 above)
Comment (Kneedler) [reply code/index.html]: Wow! Thank you, I had totally missed that. And also thank you for giving me tip on the tabindex! And thank you for your kind words!!

**Pull request closed without merging**